components/UserList.tsx

```diff
@@ -1,31 +1,45 @@
 import React from 'react'
-import { StyleSheet, ScrollView } from "react-native"
-import { connect } from "react-redux"
-import User from "../components/User"
+import { StyleSheet, ScrollView, Text, View } from 'react-native'
+import { connect } from 'react-redux'
+import User from '../components/User'
 
 const UserList = props => {
-    const userList = props.allUsers.map(user => {
-        return (
-            <User key={user.id} user={user} />
-        )
-    })
-    return (
-        <ScrollView style={styles.users}>
-            {userList}
-        </ScrollView>
-    )
+  const sentHangoutRequestEmails = props.sentHangoutRequests.map(
```

Comment:
replace user listing with message that their hangout request is pending that user's response if a request has been sent.